### PR TITLE
setup: add TF as extra-requirement

### PR DESCRIPTION
**Pull request by dimidd**
This would allow the user to specify explicitly which version of TF
is needed. E.g. `finetune[tf_gpu]`.

See this discussion for details:
https://github.com/tensorflow/tensorflow/issues/7166

Signed-off-by: Dimid Duchovny <dimidd@gmail.com>

### CODE REVIEW

**Comment by madisonmay** [comment]
This is a valid suggestion.  Would you mind adding a ">=1.11.0" requirement to those extras_require lines?  We're taking advantage of a tf.Estimator distribution strategy that wasn't present prior to 1.11.

**Comment by madisonmay** [comment]
Again, thanks for the PR @dimidd!

**Comment by dimidd** [comment]
Sure, I'll add it. Thanks for your work on finetune :)

**Comment by madisonmay** [comment]
Great! LGTM.
